simon_says/alphabot/scripts/PCA9685.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+
+import time
+import math
+import smbus
+
+# ============================================================================
+# Raspi PCA9685 16-Channel PWM Servo Driver
```

Comment:
This file needs proper acknowledgement. @wzli do you have a source?

Reply:
I have added the source. @wzli you do not need to do anything :)